### PR TITLE
Minigrid heatmaps reproduction, updated README, refactoring

### PR DESCRIPTION
**Pull request by orrp**
1. Ensure that Figure 7 from the paper (minigrid heatmaps) reproduces. Added explanation to README on how to reproduce it.
2. Removed some files and added their contents to utils.py https://github.com/micahcarroll/uniMASK/commit/d2970a3671a8c0787799b8bee4ed53d7398f7c7b

### CODE REVIEW

File: uniMASK/sweeps/medium_maze_sweep_10.yaml [hunk 1]
```diff
@@ -1,89 +0,0 @@
-program: train.py
```
Comment (micahcarroll) [inline uniMASK/sweeps/medium_maze_sweep_10.yaml:1]: @orrp  I've removed all `.sh` files and sweep `.yaml` files relating to Maze that are not necessary to reproduce the paper. Have you done that too for minigrid? If not, probably worth doing a tiny cleanup of that kind there

File: uniMASK/transformer.py [hunk 2]
```diff
@@ -84,7 +84,7 @@ def validate_training_params(tp):
     #
     #     if self.state_normalization:
     #         # Normalize (at network time, so don't have to worry about doing it at batch and eval time)
-    #         # TODO: add flag for state normalization, and add state normalization to flexiBiT too
+    #         # TODO: add flag for state normalization, and add state normalization to FB too
```
Comment (micahcarroll) [inline uniMASK/transformer.py:87]: If you ended up switching FB to UM, maybe we should do that here too

**Review by micahcarroll — APPROVED** [review]
LGTM with v minor comments! Thanks Orr!

File: uniMASK/transformer.py (hunk 2 above)
Comment (orrp) [reply uniMASK/transformer.py:87]: I did not do that, just flagged that one time. We can do this separate from this PR?

File: uniMASK/sweeps/medium_maze_sweep_10.yaml (hunk 1 above)
Comment (orrp) [reply uniMASK/sweeps/medium_maze_sweep_10.yaml:1]: done https://github.com/micahcarroll/uniMASK/pull/3/commits/a9fd1fd70b5d25d55b95fb482f053f1491c286dd

File: uniMASK/transformer.py (hunk 2 above)
Comment (micahcarroll) [reply uniMASK/transformer.py:87]: Sure yeah

**Pull request merged**